dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -366,6 +366,20 @@ PROP_SET_ERROR.useraction=Verify that the value that is specified is valid for t
 # 8060 deleted
 # 8065 deleted
 
+# 8066E
+# {0}         The invalid attribute value
+# {1}         The allowed attribute values
+# Do not translate: identifyException, as
+8066E_IDENTIFY_EXCEPTION_INVALID_TARGET=DSRA8066E: Invalid value ''{0}'' specified for the identifyException ''as'' attribute. Allowed values are: {1}
```

Comment:
Invalid value ''{0}'' specified for the identifyException ''as'' attribute.
--->
The  ''{0}'' invalid value was specified for the ''as'' attribute on the identifyException element.